client/api_object.go

```diff
@@ -1265,3 +1267,54 @@ func (c *client) ListObjectsByObjectID(ctx context.Context, objectIds []uint64,
 
 	return objects, nil
 }
+
+// ListObjectPolicies list object policies by object info and action type
+func (c *client) ListObjectPolicies(ctx context.Context, objectName, bucketName string, actionType uint32, opts types.ListObjectPoliciesOptions) (types.ListObjectPoliciesResponse, error) {
+	params := url.Values{}
+	params.Set("object-policies", "")
+	params.Set("start-after", opts.StartAfter)
+	params.Set("limit", strconv.FormatInt(opts.Limit, 10))
+	params.Set("action-type", strconv.FormatUint(uint64(actionType), 10))
+
+	reqMeta := requestMeta{
+		urlValues:     params,
+		bucketName:    bucketName,
+		objectName:    objectName,
+		contentSHA256: types.EmptyStringSHA256,
+	}
+
+	sendOpt := sendOptions{
+		method:           http.MethodGet,
+		disableCloseBody: true,
+	}
+
+	endpoint, err := c.getEndpointByOpt(opts.EndPointOptions)
+	if err != nil {
+		log.Error().Msg(fmt.Sprintf("get endpoint by option failed %s", err.Error()))
+		return types.ListObjectPoliciesResponse{}, err
+	}
+
+	resp, err := c.sendReq(ctx, reqMeta, &sendOpt, endpoint)
+	if err != nil {
+		return types.ListObjectPoliciesResponse{}, err
+	}
+	defer utils.CloseResponse(resp)
+
+	// unmarshal the json content from response body
+	buf := new(strings.Builder)
+	_, err = io.Copy(buf, resp.Body)
+	if err != nil {
+		log.Error().Msgf("the list object policies in bucket name:%s, object name:%s failed: %s", bucketName, objectName, err.Error())
```

Comment:
the log is not so useful

Reply:
we can use this err to locate specific object info and find where is the error